tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/Checks/ProcessBasicChecksTests.cs

```diff
@@ -172,15 +173,15 @@ public async Task Working()
                 standardOutput.Should().Contain(ProfilerVersion(TracerConstants.AssemblyVersion));
             }
 
-            standardOutput.Should().NotContainAny(
+            standardOutput.Should().NotContain(
```

Comment:
The additional arguments of `NotContain` are not doing what you think they're doing 😉 
```suggestion
            standardOutput.Should().NotContainAny(
```